src/Common/tests/System/Security/Cryptography/AlgorithmImplementations/AES/AesContractTests.cs

```diff
@@ -59,12 +59,16 @@ public static void LegalKeySizes()
         }
 
         [Theory]
-        [InlineData(64)]        // too small
-        [InlineData(129)]       // in valid range but not valid increment
-        [InlineData(384)]       // too large
-        [InlineData(536870928)] // number of bits overflows and wraps around to a valid size
-        public static void InvalidKeySizes(int invalidKeySize)
+        [InlineData(64, false)]        // too small
```

Comment:
There seems to be a way of doing this as I just found out in the mentioned issue but will skip it for this PR